### PR TITLE
Replace '/' with '\' for powershell in windows alerting integTest

### PR DESCRIPTION
**Pull request by peterzhuamazon**
*Issue #, if available:*
https://github.com/opendistro-for-elasticsearch/opendistro-infra/issues/251

*Description of changes:*
This PR is to replace '/' with '\\' for powershell in windows alerting integTest

*Test Results:*
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/277280437

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/staging-build-windows.yml [hunk 1]
```diff
@@ -280,7 +280,7 @@ jobs:
           echo "running tests"
           cd ..\alerting\alerting
           dir
-          ../gradlew.bat integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest -Dhttps=true -Duser=admin -Dpassword=admin
+          ..\gradlew.bat integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest -Dhttps=true -Duser=admin -Dpassword=admin
```
Comment (camerski) [inline .github/workflows/staging-build-windows.yml:283]: Line 116 has spaces after the "-D"; this line does not. Looking at the failed test run, it seems like you need the spaces.

**Review by camerski — CHANGES_REQUESTED** [review]

File: .github/workflows/staging-build-windows.yml (hunk 1 above)
Comment (camerski) [reply .github/workflows/staging-build-windows.yml:283]: Looks like your latest commit fixes this. Disregard.

Comment (peterzhuamazon) [reply .github/workflows/staging-build-windows.yml:283]: checking

Comment (peterzhuamazon) [reply .github/workflows/staging-build-windows.yml]: fixed and worked, please review.

**Review by gaiksaya — APPROVED** [review]
Please update the PR description with all changes done (-D<space>). Would be easy to catch any issues in future

**Review by camerski — APPROVED** [review]

**Pull request merged**